File: WordPress/Classes/ViewRelated/Blog/Style/WPStyleGuide+Blog.m
//
//  WPStyleGuide+Blog.m
//  WordPress
//
//  Created by Will Kwon on 10/7/15.
//  Copyright © 2015 WordPress. All rights reserved.
//

#import "WPStyleGuide+Blog.h"

@implementation WPStyleGuide (Blog)

+ (void)configureTableViewBlogCell:(UITableViewCell *)cell
{
    [self configureTableViewCell:cell];
    cell.detailTextLabel.font = [self subtitleFont];
    cell.detailTextLabel.textColor = [self greyDarken10];
    cell.backgroundColor = [self lightGrey];
}

@end

File: WordPress/Classes/ViewRelated/Blog/WPStyleGuide+Blog.swift
import Foundation
import WordPressShared

extension WPStyleGuide {
    @objc public class func configureTableViewBlogCell(_ cell: UITableViewCell) {
        cell.textLabel?.font = tableviewTextFont()
        cell.textLabel?.sizeToFit()
        cell.textLabel?.textColor = .text

        cell.detailTextLabel?.font = self.subtitleFont()
        cell.detailTextLabel?.sizeToFit()
        cell.detailTextLabel?.textColor = .textSubtle

        cell.imageView?.layer.borderColor = UIColor.divider.cgColor
        cell.imageView?.layer.borderWidth = .hairlineBorderWidth
        cell.imageView?.tintColor = .listIcon

        cell.backgroundColor = UIColor.listForeground
    }

    @objc public class func configureCellForLogin(_ cell: WPBlogTableViewCell) {
        cell.textLabel?.textColor = .text
        cell.detailTextLabel?.textColor = .textSubtle

        let fontSize = UIFont.preferredFont(forTextStyle: .subheadline).pointSize
        cell.textLabel?.font = UIFont.systemFont(ofSize: fontSize, weight: .medium)
        cell.detailTextLabel?.font = UIFont.systemFont(ofSize: fontSize, weight: .regular)

        cell.imageView?.tintColor = .listIcon

        cell.selectionStyle = .none
        cell.backgroundColor = .basicBackground
    }

}
